proposals/0304-structured-concurrency.md

```diff
@@ -905,9 +906,9 @@ static func withTaskGroup<ChildTaskResult: Sendable, GroupResult>(
 ///   - once the `withTaskGroup` returns the group is guaranteed to be empty.
 /// - if the body throws:
 ///   - all tasks remaining in the group will be automatically cancelled.
-static func withThrowingTaskGroup<ChildTaskResult: Sendable, GroupResult>(
+public func withThrowingTaskGroup<ChildTaskResult: Sendable, GroupResult>(
```

Comment:
Thanks for removing the static, let's not add public to things though -- it is done inconsistently, say all spawn et all are public too of course but the proposal text misses that.

Let's please not  add public in the proposal snippets.